File: ServerFacade/src/main/java/Request/JoinGameRequest.java
package Request;

public class JoinGameRequest {
  private String userId;
  private String code;

  public JoinGameRequest(String userId, String code) {
    this.userId = userId;
    this.code = code;
  }

  public String getUserId() {
    return userId;
  }

  public String getCode() {
    return code;
  }
}

File: FrontEnd/lib/Request/JoinGameRequest.dart
import 'package:json_annotation/json_annotation.dart';

part 'JoinGameRequest.g.dart';

@JsonSerializable()

class JoinGameRequest {
  String userId;
  String code;

  JoinGameRequest(this.userId, this.code);

  factory JoinGameRequest.fromJson(Map<String, dynamic> json) => _$JoinGameRequestFromJson(json);

  Map<String, dynamic> toJson() => _$JoinGameRequestToJson(this);
}